pypeman/nodes.py

```diff
@@ -637,9 +646,59 @@ def process(self, msg):
             file_.write(msg.payload)
         if self.safe_file:
             os.rename(dest, old_file)
+        if self.create_valid_file:
+            validation_path = Path(old_file).with_suffix(self.validation_extension)
+            validation_path.touch()
         return msg
 
 
+class FileMover(BaseNode):
+    """
+    Move file
+
+    Used to store files at another place
+    params:
+    dest_fpath : path of the destination folder (if it don't exists it creates it)
+    """
+    def __init__(self, dest_path, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dest_path = dest_path
+        if not os.path.exists(dest_path):
+            os.makedirs(dest_path)
+
+    def process(self, msg):
+        dest_fpath = os.path.join(self.dest_path, msg.meta["filename"])
+        logger.info("move file %s to dest %s" % (msg.meta["filepath"], dest_fpath))
+        shutil.move(msg.meta["filepath"], dest_fpath)
+        msg.meta["filepath"] = dest_fpath
+        return msg
+
+
+class FileCleaner(BaseNode):
+    """
+    Delete a file and/or all metafiles in a directory with same basename but
+    with a given extension
+
+    param:
+    extensions_to_rm => list of all extensions to rm
+    msg.meta["filepath"]
+    """
+    def __init__(self, extensions_to_rm=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.extensions_to_rm = extensions_to_rm
```

Comment:
you might give one example in the helptext to clearly show whether extionsion is with or without the dot (`.`)
mostly it is with the dot.

can close